src/modules/helpchan.ts

```diff
@@ -631,7 +637,7 @@ Remaining: ${channels}`);
 		const updateLoop = async () => {
 			const channel = this.getOutdatedChannels(guild)[0];
 			if (channel) {
-				this.updateHelpChannelTopicsTimeout = this.client.setTimeout(
+				this.updateHelpChannelTopicsTimeout = setTimeout(
```

Comment:
Not sure if this is right, as I don't know how `client.setTimeout` might've been different from the global one

Reply:
`client.setTimeout` only differs to the global when the bot disconnects from Discord - it's a handy feature discord.js has, but not that needed; imo we should just keep it as is.

Reply:
Had a peek at the code around this and I agree, it's fine as-is.